### PR TITLE
Moved web_content/animated into web_content/removed

### PR DESCRIPTION
**Pull request by veghdev**
Keep, until we measure test coverage.

### CODE REVIEW

**Comment by simzer** [comment]
shouldn't we just move out the folder from web_content?

File: test/integration/test_cases/test_cases.json [hunk 1]
```diff
@@ -1326,142 +1326,142 @@
                 "7f0cd74"
             ]
         },
-        "web_content/animated/composition_comparison_pie_coxcomb_column_2dis_2con": {
+        "web_content/removed/animated//composition_comparison_pie_coxcomb_column_2dis_2con": {
```
Comment (simzer) [inline test/integration/test_cases/test_cases.json:1329]: there is an unnecessary slash in the path

**Review by simzer — APPROVED** [review]

**Comment by veghdev** [comment]
> shouldn't we just move out the folder from web_content?

maybe. where web_content_removed/... ?

**Comment by simzer** [comment]
it's ok for now, then we should tidy the test_case folder in the near future

**Review by schaumb — APPROVED** [review]

**Pull request merged**